Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.yml

```diff
@@ -379,7 +328,30 @@ script:
       description: The number of engines that positively detected the indicator as
         malicious.
       type: string
-  dockerimage: demisto/python3:3.9.8.24399
+  - arguments:
+    - description: The value to scan
+      name: value
+      required: true
+    - auto: PREDEFINED
+      default: true
+      defaultValue: active
+      description: You can choose between passive and active scanning. Passive scans
+        fetch data without reaching out directly to the indicator, including performing
+        WHOIS and DNS requests. Active scans are more noisy; we'll do a quick port
+        scan and reach out to the indicator with a web browser.
+      name: scan_type
+      predefined:
+      - active
+      - passiv
+    description: Scan an indicator (IP/URL/Domain)
+    name: pulsedive-scan
+  - arguments:
+    - description: QID recieved from scan command
+      name: qid
+      required: true
+    description: Retrieve the Result
+    name: pulsedive-scan-result
+  dockerimage: demisto/python3:3.10.4.27798
```

Comment:
context outputs are missing for both commands

https://xsoar.pan.dev/docs/integrations/context-and-outputs